### ISSUE TITLE
"Registration open" when it's not

### ISSUE THREAD

**Issue by estensen** [webkom/lego-webapp#1424]
All events in https://abakus.no/events are labeled as "Påmelding åpen!", even when they're more than a week away

<img width="1058" alt="screenshot 2019-01-30 14 57 55" src="https://user-images.githubusercontent.com/9142800/51986002-80c05880-249f-11e9-80cf-8c31ee46a1c8.png">

**Comment by SmithPeder** [comment]
Yeah 👎 . I was aware of this when I pushed this code 💯 The code is actually 10/10, the problem is that a lot of events that are far, far off have no pool, that is to say, a registration group. This is due to all events being entered into the calendar at the same time with no more info than the `startTime`. So, in essence, the front-end has no way of checking if this is an event that is actually open to everyone, or just an event that is not filled in properly yet. 

I'm working on some serializer changes that will return the status of an event, so the ones that create events can mark the event as 'TBA' or something.
